rmgweb/database/templates/solvationData.html

```diff
@@ -32,9 +47,16 @@ <h3>Solvent Name: {{ solventDataInfo.0 }}</h3>
 <P>
 {% endif %}
 
-<h2>Solute Structure</h2>
+<h2>Solute Info</h2>
+
+<h3>Solute Structure</h3>
 <p>
-{{ structure|safe }}
+{% if soluteStructure == '' %}
+No structure given.
+{% else %}
+{{soluteStructure|safe}}
```

Comment:
`{{ soluteStructure|safe }}` instead?

Reply:
I added this as a fixup commit here fa0a21a. We can autosquash this fixup commit before merging.